src/components/ProjectVersion/index.jsx

```diff
@@ -1,19 +1,178 @@
-import React from "react";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
 import styled from "styled-components";
 
+import BottomNavigator from "../shared/BottomNavigator";
 import Title from "../shared/Title";
 import Select from "../shared/Select";
-import BottomNavigator from "../shared/BottomNavigator";
+import Modal from "../shared/Modal";
+import Loading from "../shared/Loading";
+import { getPages } from "../../services/pages";
+import convertLabel from "../utils/convertLabel";
+
+import useProjectVersionStore from "../../../store/projectVersion";
+import usePageListStore from "../../../store/projectPage";
+import usePageStatusStore from "../../../store/projectInit";
```

Comment:
P1: 동일한 경로에서 사용되는 컴포넌트, Store 간에 묶어줘도 좋을 것 같습니다 👍 

```suggestion
import { BottomNavigator, Title, Select, Modal, Loading } from "../shared";
import { getPages, convertLabel } from "../../services";
import { useProjectVersionStore, usePageListStore, usePageStatusStore } from "../../../store";
```